src/Microsoft.Azure.Functions.ExtensionBundle/extensions.json

```
@@ -98,6 +98,18 @@
       "orchestrationclient"
     ]
   },
+  {
+    "id": "Microsoft.Azure.WebJobs.Extensions.DurableTask.AzureManaged",
+    "majorVersion": "0",

```

Update to `"Version": "0.4.1-alpha"
```suggestion
"Version": "0.4.1-alpha",
```